src/libstd/sys/unix/process/process_common.rs

```diff
@@ -149,7 +149,7 @@ impl Command {
         &mut self.closures
     }
 
-    pub fn before_exec(&mut self,
+    pub unsafe fn pre_exec(&mut self,
                        f: Box<dyn FnMut() -> io::Result<()> + Send + Sync>) {
```

Comment:
Er sorry, I meant that the indentation here is a little off

Reply:
Ah I see. Done.